wpilibc/src/main/native/cpp/GenericHID.cpp

```diff
@@ -31,7 +33,7 @@ bool GenericHID::GetRawButtonReleased(int button) {
 }
 
 double GenericHID::GetRawAxis(int axis) const {
-  return m_ds->GetStickAxis(m_port, axis);
+  return ApplyDeadband(m_ds->GetStickAxis(m_port, axis), m_deadband);
```

Comment:
Commonly teams would want to apply different deadbands to different axes, for example the twist on a joystick, or the triggers on a gamepad, will likely have different deadzones to the X/Y axes.

It seems that this would not be useful to teams that want to utilise these other axes.

Reply:
That use case isn't supported by the current drive class implementation either. The same deadband in the drive class applies to all axes.

Reply:
Sure, but it looks like this would apply regardless of whether you're using the axis for driving.